src/transformers/models/mpt/modeling_mpt.py

```diff
@@ -251,178 +251,107 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
             return mpt_gelu_forward(x)
 
 
-# Copied from transformers.models.bloom.modeling_bloom.BloomAttention with Bloom->Mpt
+def scaled_multihead_dot_product_attention(query, key, value, n_heads, past_key_value=None, softmax_scale=None, attn_bias=None, key_padding_mask=None, is_causal=False, dropout_p=0.0, training=False, needs_weights=False, multiquery=False):
```

Comment:
wll probably use the same format as `umt5` or other encoder decoder models where we use the same attention! Should be similar